apps/condo/domains/banking/schema/BankCategory.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Generated by `createschema banking.BankCategory 'name:Text;'`
+ */
+
+const { Text, Relationship, Integer, Select, Checkbox, DateTimeUtc, CalendarDay, Decimal, Password, File, Url } = require('@keystonejs/fields')
+const { Json, LocalizedText } = require('@condo/keystone/fields')
+const { GQLListSchema } = require('@condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@condo/keystone/plugins')
+const access = require('@condo/domains/banking/access/BankCategory')
+
+
+const BankCategory = new GQLListSchema('BankCategory', {
+    schemaDoc: 'Expenses category, that will be associated with transactions and contractors',
+    fields: {
+
+        name: {
+            schemaDoc: 'Key for i18n',
+            type: LocalizedText,
+            isRequired: true,
+            template: 'banking.category.*.name',
```

Comment:
Does it match with `ADR15` translation agreement?

Reply:
It is not confirmed yet)

Reply:
Anyway I can't actually see any translations related to this template

Reply:
Discussed, it does confirms to standard.
We does not have a list of categories yet, but it supposed to be soon